<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>584</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
	background-color:rgb(242, 234, 255);
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical{
	border:none;
	width: 14px;
	background-color: rgb(107, 100, 124);
	border-radius:0px;
}

QScrollBar::handle:vertical{
	min-height:30px;
	background-color: rgb(107, 100, 124,150);
	border-radius:7px;
}

QScrollBar::handle:vertical:hover{
	min-height:30px;
	background-color: rgb(107, 100, 124,50);
	border-radius:7px;
}

QScrollBar::handle:vertical:pressed{
	min-height:30px;
	background-color: rgb(107, 100, 124,200,100);
	border-radius:7px;
}


</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-341</y>
        <width>808</width>
        <height>912</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>900</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
	
	background-color:rgb(242, 234, 255);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="logo">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>-1</y>
            <width>375</width>
            <height>115</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	padding: 10;
	color: rgb(0, 85, 127);
	font: 38pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; color:#00557f;&quot;&gt;Recommended &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; color:#00557f;&quot;&gt;Movies&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="movieSearchBox">
          <property name="geometry">
           <rect>
            <x>25</x>
            <y>123</y>
            <width>561</width>
            <height>53</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	padding:15px;
	font: 14pt &quot;Nasalization Rg&quot;;
	border:none;
	border-radius:20px;
	background-color: rgb(199, 176, 255,100);
}
QLineEdit:hover{
	background-color: rgb(199, 176, 255,50);
}
QLineEdit:focus{
	background-color: rgb(199, 176, 255,150);
}</string>
          </property>
          <property name="placeholderText">
           <string>üîç Search for a movie...</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="searchButton">
          <property name="geometry">
           <rect>
            <x>594</x>
            <y>124</y>
            <width>51</width>
            <height>50</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	border-radius:25px;
	background-color: rgb(145, 177, 194,120);
	font: 14pt &quot;Nasalization Rg&quot;;
}
QPushButton:hover{
	background-color: rgb(145, 177, 194,100);
}
QPushButton:pressed{
	background-color: rgb(145, 177, 194,160);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Pictures/res.qrc">
            <normaloff>:/main/srch.png</normaloff>:/main/srch.png</iconset>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>22</x>
            <y>208</y>
            <width>822</width>
            <height>684</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="grid">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="7" column="1">
            <widget class="QPushButton" name="movieButton5">
             <property name="text">
              <string>Book Now</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="movieButton3">
             <property name="text">
              <string>Book Now</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QPushButton" name="movieButton6">
             <property name="text">
              <string>Book Now</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="movieButton2">
             <property name="text">
              <string>Book Now</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="movieLabel6">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="movieLabel3">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="movieLabel4">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="movieLabel2">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="movieButton1">
             <property name="text">
              <string>Book Now</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="movieLabel1">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="movieName2">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="movieName1">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="movieName3">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QPushButton" name="movieButton4">
             <property name="text">
              <string>Book Now</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLabel" name="movieName6">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="movieName4">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="movieLabel5">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="movieName5">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="backButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>51</width>
            <height>50</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	border-radius:25px;
	background-color: rgb(145, 177, 194,120);
	font: 14pt &quot;Nasalization Rg&quot;;
}
QPushButton:hover{
	background-color: rgb(145, 177, 194,100);
}
QPushButton:pressed{
	background-color: rgb(145, 177, 194,160);
}</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Pictures/res.qrc"/>
 </resources>
 <connections/>
</ui>
